### ISSUE TITLE
Confusion with batches shown in progress bar

### ISSUE THREAD

**Issue by cbirchsy** [kuleshov-group/caduceus#47]
Hi, I noticed a strange behaviour when running the pre-training command with these options:

```
dataset.batch_size=1
dataset.batch_size_eval=1
train.global_batch_size=1 
trainer.max_steps=1000
trainer.limit_val_batches=1000 
trainer.limit_train_batches=1000 
```
The lightning progress bar looks like this e.g:

`Epoch 0:  34% 1020/3000 [01:04<02:05, 15.73it/s, loss=1.2, v_num=1024]`

showing 3x more batches in the epoch than specified and training exceeds max_steps and stops at 3000. Meanwhile the validation loader shows the correct amount e.g:

`Validation DataLoader 1: 100% 1000/1000 [00:18<00:00, 55.55it/s]`

Curious why the 3x appears in the progress bar during training?

**Comment by cbirchsy** [comment]
Nevermind, looks like the number of batches shown in progress bar for an epoch is train_batches+2*val_batches

**Comment by yair-schiff** [comment]
Hi @cbirchsy , sorry for the delayed response. Yes I think this version of the lightning in our env adds validation/test dataloader to the progress bar as well. You'll notice that as this trains when you reach validation another bar shows up and both the validation progress bar and the main epoch one progress.

Glad you were able to figure this out.
